docs/content/posts/2023-05-19-release-1.16.2.md

```diff
@@ -0,0 +1,215 @@
+---
+title:  "Apache Flink 1.16.2 Release Announcement"
+date: "2023-05-19T13:00:00.000Z"
+aliases:
+- /news/2023/05/19/release-1.16.2.html
+authors:
+- reswqa:
+  name: "Weijie Guo"
+  twitter: "WeijieGuo12"
+---
+
+The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.16 series.
+
+This release includes 104 bug fixes, vulnerability fixes, and minor improvements for Flink 1.16.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12352765).
+
+We highly recommend all users upgrade to Flink 1.16.2.
+
+# Release Artifacts
+
+## Maven Dependencies
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.16.2</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>1.16.2</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>1.16.2</version>
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink?tab=tags&page=1&name=1.16.2) (official images)
```

Comment:
Even I left a comment to approve before, I just realize that this link would not point to the real `1.16.2` related tags. I think the actual link should be `https://hub.docker.com/_/flink/tags?page=1&name=1.16.2`

Reply:
Ah, good catch. Does this mean that there is already a problem with the release note in `1.16.1`. It's also possible that the link was still correct at the time. 🤔

Reply:
Yes, and I already fix these links in https://github.com/apache/flink-web/pull/651